docs/stories/index.js

```diff
@@ -0,0 +1,47 @@
+import { addStory } from 'glutenberg';
```

Comment:
Will we have more than one file in this folder? Should we start as `docs/stories.js` to avoid prematurely nesting directories?